### PR TITLE
RD-1969 Get rid of events.id and logs.id columns

### PR DESCRIPTION
**Pull request by mateumann**
Let's get rid of `events.id` and `logs.id` which were always empty anyway.

### CODE REVIEW

**Comment by tehasdf** [comment]
we note that inserting logs/events never even populated the id field anyway, https://github.com/cloudify-cosmo/cloudify-manager/blob/2313bd844ef74c6bddb177d6d29ab131de47d973/amqp-postgres/amqp_postgres/postgres_publisher.py#L21

**Review by tehasdf — DISMISSED** [review]

File: rest-service/manager_rest/storage/resource_models.py [hunk 1]
```diff
@@ -1265,7 +1265,9 @@ class Event(SQLResourceBase):
             name='events__one_fk_not_null'
         ),
     )
-    id = db.Column(db.Text, index=True, default=lambda: str(uuid.uuid4()))
+    is_id_unique = False
+
+    id = None
```
Comment (tehasdf) [inline rest-service/manager_rest/storage/resource_models.py:1270]: this is just to override the parent class attribute

Comment (geokala) [reply rest-service/manager_rest/storage/resource_models.py:1270]: That should maybe be a comment in the code itself?

Comment (mateumann) [reply rest-service/manager_rest/storage/resource_models.py:1270]: I'll add that comment in next PR

Comment (mateumann) [reply rest-service/manager_rest/storage/resource_models.py]: (added in this one)

**Review by tehasdf — APPROVED** [review]

**Pull request merged**